theme/styles/settings/_vars.scss

```diff
@@ -30,4 +29,3 @@ $cart-checkout-container-height: 3.5625rem;
 $cart-checkout-container-height-lg: 3.875rem;
 
 $border-offset: 0.125rem;
```

Comment:
Now using `border-offset` var for the News Feed Items hover styling positioning offset